docs/synthetics/status-pages.md

```diff
@@ -5,51 +5,56 @@ Using Sematext Synthetics you can create public and private status pages to shar
 
 * Communicate the status of your services to your customers
 * Share status updates with your customers during planned maintenance or outages
-* Create a dashboard of service status to share across internal teams
+* Create a service status dashboard to share across internal teams
```

Comment:
dashboard ==> page?

Reply:
I wasn't sure about this one either, but left it as dashboard since the status pages give an overview of multiple monitors, incidents, and have charts on them, so they are kind-of sort-of dashboards. Dashboard also sounds fancier in these starting bullet points as a selling point, that could also be why it's there  ¯\_(ツ)_/¯

Reply:
Sure, let's let status dashboard live then.